src/enrollments/enrollments.controller.ts

```diff
@@ -28,4 +33,11 @@ export class EnrollmentsController {
   ): Promise<void> {
     return this.enrollmentsService.removeStudent(user.id, deleteEnrollDto);
   }
+
+  @Roles(EUserType.TEACHER)
+  @Get("students/:classroomId")
+  async getStudentsForClassroom(@Param("classroomId") classroomId: number): Promise<IEnrollment[]> {
+    const enrollments = await this.enrollmentsService.getStudentsForClassroom(classroomId);
+    return this.enrollmentSerializer.serializeMany(enrollments);
+  }
```

Comment:
I think this should be inside the classrooms controller

Reply:
This is fetching "enrolled" students for specific classroom, that's why I have decided to include it in this controller.